### PR TITLE
Receipt CSV file for HealthPartners Clinical collections showing incorrect vial types

### PR DESCRIPTION
**Pull request by Gbarra9**
This PR addresses the following issue:

- https://github.com/episphere/connect/issues/947

Problem:
Incorrect Vial Type mapping

- https://github.com/episphere/connect/issues/947#issue-2219058752

Code Change:

Add `healthPartners` object with correct map references for tubes to `vialMapping` 

Rename `nameToKeyObj` and `keyToNameAbbreviationObj ` and update references

### CODE REVIEW

**Comment by we-ai** [comment]
Can we use `healthPartners` instead of `hPartners`?

**Comment by jhflorey** [comment]
It looks good to me

**Comment by Gbarra9** [comment]
> Can we use `healthPartners` instead of `hPartners`?

@we-ai 

I kept the property name to `hPartners`. Do you prefer to have the other abbreviated names changed? I can change `hPartners` to `healthPartners` and the other abbreviated for consistency. It will require renaming `nameToKeyObj` and`cidToHealthcareProvider` constants. I can also take care of the name references in other places of the code such as `vialMapping`

Let me know if you want the names to be changed (my initial changes for renaming are below)

**Healthcare provider abbreviation renaming:**

kpNW --> kaiserPermanenteNW
hPartners --> healthPartners
snfrdHealth --> sanfordHealth
hfHealth --> henryFordHealth
maClinic --> marshfieldClinic
kpCO --> kaiserPermanenteCO 
uChiM --> universityfChicagoMedicine
kpHI --> kaiserPermanenteHI
kpGA --> kaiserPermanenteGA

**Constants renaming:**

nameToKeyObj --> healthcareProviderToConceptIdObj 

keyToNameAbbreviationObj --> conceptIdToHealthcareProviderObj

--------------------------------------------------------------------------------------------------------------------------
Current constants with abbreviated healthcare provider property name references and values

export const nameToKeyObj = 
{
    "kpNW": 452412599,
    "hPartners" : 531629870,
    "snfrdHealth": 657167265,
    "hfHealth": 548392715,
    "maClinic": 303349821,
    "kpCO": 125001209,
    "uChiM": 809703864,
    "nci": 13,
    "kpHI": 300267574,
    "kpGA": 327912200,
    "allResults": 1000
}

export const keyToNameAbbreviationObj = {
  452412599: "kpNW",
  531629870: "healthPartners",
  657167265: "snfrdHealth",
  548392715: "hfHealth",
  303349821: "maClinic",
  125001209: "kpCO",
  809703864: "uChiM",
  13: "nci",
  300267574: "kpHI",
  327912200: "kpGA",
  1000: "allResults",
}

**Comment by we-ai** [comment]
My main concern is readability. 
Here are just my opinions for your references. When we say "health partner", the name better be `healthPartner`. When we say "KPCO", "KPHI" etc., I guess `kpco` (or kpCO), `kphi` (or kpHI) should be fine. For `uChiM`, a better name should be `universityOfChicagoMedicine` (or `uOfChicagoMedicine`, `uOfChicagoMed` etc)

**Review by jhflorey — APPROVED** [review]

File: src/shared.js [hunk 1]
```diff
@@ -1988,7 +1987,7 @@ export const locationConceptIDToLocationMap = {
   [conceptIds.nameToKeyObj.hpPN]: {
     siteSpecificLocation: 'HP Park Nicollet',
     siteAcronym: 'HP',
-    siteCode: nameToKeyObj.hPartners,
+    siteCode: `${healthProviderAbbrToConceptIdObj.healthPartners}`,
```
Comment (we-ai) [inline src/shared.js:1990]: Do we need string here?

**Review by we-ai — APPROVED** [review]
Looks good. Commented on minor things.

File: src/shared.js (hunk 1 above)
Comment (Gbarra9) [reply src/shared.js:1990]: The other siteCode values use concept Ids with string values. I had to change the date type from number to string.

**Comment by brotzmanmj** [comment]
@we-ai  and @Gbarra9  can you confirm the site naming you were discussing above is just internal to the system and has no bearing on the site names in the CSV file? Thanks

**Comment by Gbarra9** [comment]
@brotzmanmj I can confirm this will have no impact to the naming in the CSV files. The site naming changes discussed were done for code readability. The references should still be the same.

**Pull request merged**